common/static/coffee/src/discussion/discussion_filter.coffee

```diff
@@ -1,28 +0,0 @@
-class @DiscussionFilter
-
-  # TODO: this helper class duplicates functionality in DiscussionThreadListView.filterTopics
-  # for use with a very similar category dropdown in the New Post form.  The two menus' implementations
-  # should be merged into a single reusable view.
-
-  @filterDrop: (e) ->
```

Comment:
Note to reviewers: This method was only used in 1 place, so I just moved it to the place it was used (as straight JS).